constants/poolFilterList.ts

```diff
@@ -1,10 +1,18 @@
+/*
+	This array includes the pool that are hidden in the pool list
+ */
+
 export const filterList = [
 	'0xbc465ef7e2d4f37d92dda7d449b82d245d06b6a7',
-	'0xd25502dff3ea3291da50ad3b0f202db03d4eb1a6',
-	'0xee9146721a3d9e93d95ba536390008ac0df3c0d6',
-	'0xb601cecd429fe0eaf424b6d96730ac1c66937e38',
-	'0x59158a5e8e32f4e514e05d170068a0a4a5f19d71',
-	'0x2b3b1259b0be38a172a3d5cd9ea4385dba0ccf4c',
-	'0xe009380daa262b5e80b5b4ec1116a9a42f2f3150',
-	'0xadabccd58d56596bfc23d6aaa1b24df90af2e021',
+	'0xd25502dff3ea3291da50ad3b0f202db03d4eb1a6', // test
+	'0xee9146721a3d9e93d95ba536390008ac0df3c0d6', // Aelin Pool
+	'0xb601cecd429fe0eaf424b6d96730ac1c66937e38', // Pool One
+	'0x59158a5e8e32f4e514e05d170068a0a4a5f19d71', // alien
+	'0x2b3b1259b0be38a172a3d5cd9ea4385dba0ccf4c', // SELDON3
```

Comment:
seldon 3 has funds to withdraw. please make sure to only hide the test pools with 0 funds in them